ci/jobs/build_clickhouse.py

```diff
@@ -153,6 +153,10 @@ def main():
             Shell.check(
                 f"ln -sf /build/cmake/toolchain/darwin-x86_64 {current_directory}/cmake/toolchain/darwin-aarch64"
             )
+        elif build_type in (BuildTypes.AMD_TIDY, BuildTypes.ARM_TIDY):
+            run_shell("clang-tidy-cache stats", "clang-tidy-cache.py --show-stats")
+        run_shell("sccache start server", "sccache --start-server", retries=3)
```

Comment:
```suggestion
        run_shell("sccache start server", "sccache --start-server")
```

not related, but it does nothing and looks awkward

Reply:
What do you mean, do nothing? https://github.com/ClickHouse/ClickHouse/blob/c5331aa58cbc95e18f644027bdbb16cc20f9ad79/ci/jobs/build_clickhouse.py#L82 -> https://github.com/ClickHouse/ClickHouse/blob/master/ci/praktika/utils.py#L212

This is an intentional retry to mitigate 3d party issues.

Reply:
> https://github.com/ClickHouse/ClickHouse/blob/master/ci/praktika/utils.py#L385, isn't it the place that checks the exit code?
you need to pass strict=True to have this exception raised.

> https://github.com/ClickHouse/ClickHouse/pull/84674#issuecomment-3132608222 not the first time. The first time, retries were added. But the server's starting point was wrong.
it's another point of failure. sccache start command was ok here

Reply:
> it's another point of failure. sccache start command was ok here

Where? The point of failure is [cmake](https://github.com/ClickHouse/ClickHouse/actions/runs/16558568994/job/46828477220#step:5:640). The Server would attempt to start later.

> you need to pass strict=True to have this exception raised

Valid point, but rather, it would make sense to make it a `Result`.